project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/model/exception/ExceptionTransformer.kt

```diff
@@ -0,0 +1,20 @@
+package org.plan.research.minimization.plugin.model.exception
+
+import arrow.core.Option
+import arrow.core.toOption
+import org.plan.research.minimization.plugin.execution.IdeaCompilationException
+import org.plan.research.minimization.plugin.execution.exception.KotlincException
+import org.plan.research.minimization.plugin.execution.exception.KotlincException.*
+
+interface ExceptionTransformation {
```

Comment:
\+ you can provide default implementation all methods like return itself

Reply:
Yea, I thought in same situations the transformations should fail and we need to cover that case, but don't think this matters

Reply:
I think that transformers shouldn't fail (or if it fails, it should return unchanged result), but if you think it will be better to handle somehow errors and tell about it outside, I can suggest some solution